package.json

```diff
@@ -27,6 +30,9 @@
   "license": "Apache-2.0",
   "bugs": {},
   "homepage": "https://github.com/strimzi/strimzi-ui#readme",
+  "cypress-cucumber-preprocessor": {
```

Comment:
I would like the package.json to stay as (tooling) config free as possible. Does the cypress cucumber processor offer a config location parameter which we could use?